cranelift-simplejit/src/memory.rs

```diff
@@ -2,6 +2,7 @@
 use errno;
 
 #[cfg(not(feature = "selinux-fix"))]
+#[cfg(not(windows))]
```

Comment:
Would it work to combine this with the previous line, as `#[cfg(not(any(feature = "selinux-fix", windows)))]`?

Reply:
I considered it in the first place, but current approach seemed to me a little bit more readable. Will fix it.